### DISCUSSION TITLE
Voikko ei tunnista kaikkia kilauttaa-sanan taivutusmuotoja

### DISCUSSION THREAD

**Discussion by ahomansikka** [voikko/corevoikko#59] [General]
$ echo kilautan |voikkospell
W: kilautan
$ echo kilautat |voikkospell
W: kilautat
$ echo kilauttaa |voikkospell
C: kilauttaa
$ echo kilautettu |voikkospell
C: kilautettu

Ja niin edelleen.

Joukahaisessa ei ole kilauttaa-sanaa, ja Voikossa se näytetään
johdetun kilistä-sanasta. Kokeile

echo kilauttaa | voikkospell -m

Asialla näyttää olevan jotain tekemistä myös sen kanssa, mikä on
yksipersoonainen sana ja mikä ei.

Voikossahan sanojen taivutuskaavoissa on myös sanojen johtaminen.
Useimmissa tapauksissa tämä näyttää toimivan aivan oiken, mutta ei
juuri tässä.

Ongelman voi helpoiten ratkaista (kiertää?) lisäämällä kilauttaa-sana
Joukahaiseen, kuten muutkin vastaavat sanat sitä mukaa kuin niitä löytyy.

Ongelma tuntuu niin vähäiseltä, että sen takia ei kannata lähteä
erottamaan koodissa sanojen taivutusta ja johtamista.

**Comment by hatapitk** [comment]
Hei!

Hyvä huomio. Koskee myös mm. sanaa "pirautan". Tämä korjautuisi ehkä parhaiten nollaamalla persoonamuotoja rajoittava lippu ennen johdoksen muodostamista. Voin katsoa tätä ensi viikolla, ellet ehdi ensin.

**Comment by ahomansikka** [comment]
Sinä tunnet kyseisen koodin paremmin kuin minä eli jään odottamaan.

su, 2024-11-10 kello 13:02 -0800, Harri Pitkänen kirjoitti:

**Comment by hatapitk** [comment]
Nyt korjattu: dfee14c0fcdf17d12d69ae45cb503bc79da661a5
